hack/update-crds.sh

```diff
@@ -39,6 +39,11 @@ export PATH=$(dirname "$go"):$PATH
 REPO_ROOT=${BUILD_WORKSPACE_DIRECTORY}
 cd "${REPO_ROOT}"
 
+"$controllergen" \
+    crd \
+    paths=./pkg/webhook/handlers/testdata/apis/testgroup/v{1,2}/... \
+    output:crd:dir=./pkg/webhook/handlers/testdata/apis/testgroup/crds
```

Comment:
I'm not sure if I need to make a corresponding change to the local `.bazel` file to tell bazel that this script should be re-run if the `//pkg/webhook/handlers/testdata/apis/testgroup:all-srcs` change.
I think not, because `verify-crds.sh` will be run anyway and fail if the srcs change and the output will therefore change and print a diff.

And do I need to tell bazel that this script now produces additional output files?

Reply:
I think we resolved this on our call the other day?